libmscore/layout.cpp

```diff
@@ -2674,6 +2726,21 @@ void Score::getNextMeasure(LayoutContext& lc)
                                     chord->computeUp();
                                     chord->layoutStem1();   // create stems needed to calculate spacing
                                                             // stem direction can change later during beam processing
+
+                                    // if there is a two-note tremolo attached, and it is too steep,
+                                    // extend stem of one of the chords (if not cross-staff)
+                                    // or extend both stems (if cross-staff)
+                                    // this should be done after the stem lengths of two notes are both calculated
```

Comment:
I guess the problem you are having is that there is no way stem lengths can be known this early for the cross staff case, at least not if beams are involved.  Probably not for the cross-measure case either.  This is only finalized later.

While it *might* be possible to resolve these issues with some work, to me it would be acceptable to not do any such adjustment for cross-staff or cross-measure cases,and just let users contonue to need to adjust these manually.  not sure how easy it is to detect cross-measure at this point though.

Reply:
Aside: One of the issues of leaving customization of tremolos between notes up to the user is that the user can't adjust the "grow left/right" (cf. feather beams terminology) related to the tremolos (or maybe they can and I'm missing something). Although the placement algorithm works well enough for most use cases, maybe it would be appropriate to have inspector attributes for changing the angles of note tremolos for some "off cases", ... but that's another story.

Reply:
@MarcSabatella Manual adjustment can work for minim cross-staff tremolo and shorter, because you can adjust stem lengths. But for semibreve and longer there's no way of manual adjustment.

Reply:
Plus, there aren't any grips ("adjustment handles") for the tremolo strokes themselves. Inspector attributes are nice, but it's nicer to have grips ;-)

Reply:
True, I mixed up one and two note tremolos in my head here.  Anyhow, I assume it's still the case that this isn't going to work here, you'd need to do this at the point where the stem length is actually calculated during cross-staff layout.

Reply:
Stem lengths are correct now, it's `stemPos()` (or `stem()->pagePos()`) that's wrong. But thanks for replying!

Reply:
Stem directions also are not known for sure until cross staff processing, and thus, stem positions too.